content/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Data Together"
+tagline: "Communities Stewarding Data Together"
+date: 2017-09-10T18:00:00-04:00
+draft: false
+---
+
+<div class="signup container">
+  <div class="row">
+    <div class="text col-md-6 col-sm-12 col-xs-12">
+      <p class="sub-emphasis">Data Together is a new model for distributed, community-driven stewardship of data. This model shows how groups of people, companies, institutions, NGOs, and governments can use decentralized and peer-to-peer web infrastructure to access, discover, verify, and preserve data they care about. We hope to build a future in which networks of collaborators make their data accessible to their peers, immediately discoverable, easily verifiable, and robustly preserved.</p>
```

Comment:
Minor grammatical issue: the referent of "immediately discoverable" and the tail end of that sentence strike me as ambiguous. Here are some ideas for how to avoid the problem:

- Replace the comma after "peers" with a long dash (using whichever "long dash" convention is being used throughout the project, whether that's an em-dash without spaces or an en-dash surrounded by spaces)
- Add the words "in ways that are" before "immediately"
- Replace the comma after "peers" with "and the data is" (or "and the data are" if we're following the tradition that "data" is considered plural)